### DISCUSSION TITLE
Please help me resolve the problem of "account flagged"

### DISCUSSION THREAD

**Discussion by woniu7** [community/community#142345] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

It's my friend's github account:  wangjc0216
Origin notvisible:  https://github.com/orgs/community/discussions/142213
My account is used since 7 years ago. But my account has been flagged for a some time . I made a ticket but got no reply : https://support.github.com/ticket/personal/0/3014988
Now I can't share my repo with my fellow because they can't see any information about my account , and I can't loggin other website with github Oauth.
Please github worker help me unlock my account . Thanks a lot !

@LiteBrite82 @queenofcorgis Could you help me take a look ? I saw you reply similar question in https://github.com/orgs/community/discussions/142115 and https://github.com/orgs/community/discussions/141780 . I really need yours help .Your help is very import for me . Thank you very much !

**Comment by LakinduVirajith** [comment]
It seems like your account has been flagged, and you're facing issues accessing your repositories and logging in through GitHub OAuth. Since you've already submitted a support ticket but haven't received a response yet, I recommend following up with GitHub Support by replying to your ticket email to get an update on the status.

As for reaching out to @LiteBrite82 and @queenofcorgis, it’s great that you’ve noticed their involvement in similar discussions. However, it’s best to wait for GitHub Support to resolve this issue, as they’re the ones who can unlock your account.

**Reply by woniu7** [reply]
Thank you for your reply,but my friend submit the ticket 3 weeks ago but got no reply. Could you help him accelerate the ticket review? Or help check the correction of the ticket.Thank you very much. 🌹 @LiteBrite82 @queenofcorgis

**Reply by queenofcorgis** [reply]
@LakinduVirajith 
We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @woniu7, 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
